hasura.planx.uk/metadata/tables.yaml

```diff
@@ -1340,6 +1340,10 @@
 - table:
     name: team_integrations
     schema: public
+  object_relationships:
+    - name: team
+      using:
+        foreign_key_constraint_on: team_id
```

Comment:
Adding this relationship means that `team_integrations` will be deleted via cascade when their team is deleted.